### DISCUSSION TITLE
Rakugaki Showtime: fully playable, some graphic glitches

### DISCUSSION THREAD

**Discussion by ritzable** [kootstra-rene/enge-js#32] [General]
Game:
Rakugaki Showtime, JP Region

Date tested: 
2022-05-17

GameCode: SLPM-86272

Comments:
Repo is probably dead BUT, i tried the first JP region game on here, one that i like.
I've played through the whole game and, it's fully playable.
Only minor bug, minor graphical glitches in the game with menus, and gameplay.
other then that the game is fine!

Screenshots:
![Screenshot 2022-05-17 7 38 24 AM](https://user-images.githubusercontent.com/78597960/168805781-a4fb1256-383c-4208-864a-c4db9bb6da7b.png)
![Screenshot 2022-05-17 7 37 44 AM](https://user-images.githubusercontent.com/78597960/168805786-47ece4d9-2108-47e2-afa6-be4218baf7dd.png)
![Screenshot 2022-05-17 7 37 23 AM](https://user-images.githubusercontent.com/78597960/168805788-60c8f2db-6c9d-4ba2-93b5-d59ec87e3cd0.png)
![Screenshot 2022-05-17 7 37 12 AM](https://user-images.githubusercontent.com/78597960/168805789-7c1f9be0-ced2-456a-9a4c-466146aa1bf9.png)
![Screenshot 2022-05-17 7 36 13 AM](https://user-images.githubusercontent.com/78597960/168805791-36e6cb5e-c988-451b-8641-2ef7d37ea1b6.png)

**Comment by kootstra-rene** [comment]
Thanks for the info and the addition to the compatibility list. Good to see that the emulator is used. 

It is not dead but at a low priority for now...
